addons/web/static/tests/core/network/rpc.test.js

```diff
@@ -129,6 +129,7 @@ test.tags("headless")(
 );
 
 test.tags("headless")("check connection aborted", async () => {
+    after(mockFetch(() => new Promise(() => {})));
```

Comment:
or should this be the default? Maybe not, I guess most of tests doing rpcs need a real mocked version of fetch